### PR TITLE
Update to latest Next.js canary and fix for custom web pages

### PR DESCRIPTION
**Pull request by lfades**

### CODE REVIEW

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**/\[...pages\]**](https://commerce-git-fix-pages.vercel.vercel.app/[...pages]) |
|:-:|
|<a href="https://commerce-git-fix-pages.vercel.vercel.app/[...pages]">Screenshot of /[...pages]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/eWfKhg1B2fZTpeSRBTSz_7d_4JU_.png">(view full size)</a> |







Commit c1db48bb665373f6fddff4766e600d59f98b58f3 (https://commerce-git-fix-pages.vercel.vercel.app).

File: pages/[...pages].tsx [hunk 1]
```diff
@@ -32,11 +37,22 @@ export async function getStaticProps({
...
 }
 
-export async function getStaticPaths() {
+export async function getStaticPaths({ locales }: GetStaticPathsContext) {
   const { pages } = await getAllPages()
+  const [invalidPaths, log] = missingLocaleInPages()
+  const paths = pages
+    .map((page) => page.url)
+    .filter((url) => {
+      if (!url || !locales) return url
```
Comment (chibicode) [inline pages/[...pages].tsx:46]: is this line correct?

Comment (lfades) [reply pages/[...pages].tsx:46]: If there's no `url`, it returns `url` and the page will be ignored (note this is happening inside `.filter()`).

If there's no `locale`, it returns the `url` of the page, the example works if i18n is disabled (I tested that multiple times 👍 )

Comment (chibicode) [reply pages/[...pages].tsx:46]: ok!

**Review by chibicode — APPROVED** [review]

**Pull request merged**